tests/quic/testQuicConnection.nim

```diff
@@ -58,18 +58,10 @@ suite "quic connection":
       )
 
   asyncTest "performs handshake":
-    let (client, server) = await performHandshake()
-    defer:
-      await client.drop()
-    defer:
-      await server.drop()
-
-    check client.handshake.isSet()
-    check server.handshake.isSet()
-
-  asyncTest "performs handshake multiple times":
-    for i in 1 .. 100:
+    for i in 1 .. 10:
       let (client, server) = await performHandshake()
+      check client.handshake.isSet()
+      check server.handshake.isSet()
       await client.drop()
       await server.drop()
```

Comment:
simplified "performs handshake" test:
- reduce count to 10 (100 takes too much time)
- combined both tests to single test"